projects/plugins/jetpack/extensions/blocks/blogroll/blogroll-item/edit.js

```diff
@@ -17,7 +17,14 @@ function BlogrollItemEdit( { className, attributes, setAttributes } ) {
 				render={ ( { open } ) => (
 					<Button variant="link" onClick={ open } style={ { padding: 0 } }>
 						<figure>
-							<img src={ icon } alt={ name } />
+							<img
+								onError={ event => {
+									event.target.src = 'https://s0.wp.com/i/webclip.png';
+									event.onerror = null;
```

Comment:
This seems like a mistake? `event` has no `onerror` prop. To prevent an infinite loop, I would do this:
I would do this

```suggestion
									if ( event.target.src !== 'https://s0.wp.com/i/webclip.png' ) {
									     event.target.src = 'https://s0.wp.com/i/webclip.png';
									}
```

Reply:
That makes sense @alshakero, great catch! Addressed 👍